simplemind/providers/ollama.py

```
@@ -36,7 +36,7 @@ def structured_client(self):
             mode=instructor.Mode.JSON,
         )
 
-    def send_conversation(self, conversation: "Conversation"):
+    def send_conversation(self, conversation: "Conversation") -> "Message":
```

Ruff

Undefined name `Conversation`

(F821)